closure/goog/array/array_test.js

```diff
@@ -281,6 +281,12 @@ function testArrayReduceOmitDeleted() {
   assertEquals(2, goog.array.reduce(a, scope.testFn, 0, scope));
 }
 
+function testArrayReducePreviousValuePatch(){
```

Comment:
The test name description should describe what this is testing. This is actually testing behavior when initialValue isn't provided.